pipelinewise/fastsync/commons/target_snowflake.py

```diff
@@ -195,21 +201,21 @@ def grant_select_on_table(self, target_schema, table_name, role, is_temporary, t
             table_dict = utils.tablename_to_dict(table_name)
             target_table = table_dict.get('table_name') if not is_temporary else table_dict.get('temp_table_name')
             sql = 'GRANT SELECT ON {}."{}" TO ROLE {}'.format(target_schema, target_table.upper(), role)
-            self.query(sql)
+            self.query(sql, query_tag_props={'schema': target_schema, 'table': table_name})
 
     # pylint: disable=unused-argument
     def grant_usage_on_schema(self, target_schema, role, to_group=False):
         # Grant role is not mandatory parameter, do nothing if not specified
         if role:
             sql = 'GRANT USAGE ON SCHEMA {} TO ROLE {}'.format(target_schema, role)
-            self.query(sql)
+            self.query(sql, query_tag_props={'schema': target_schema, 'table': None})
```

Comment:
> Given how the table value is being dealt with in the open_connection method, there is no need to mention the table here 'table': None.

Same as this.

Reply:
fixed by 26febe82084667ac7e7db47299147e0dfa284257